crates/ra_project_model/src/lib.rs

```diff
@@ -273,6 +273,16 @@ impl ProjectWorkspace {
                         };
                         let cfg_options = {
                             let mut opts = default_cfg_options.clone();
+                            for cfg in &krate.cfg {
+                                match cfg.find('=') {
+                                    None => opts.insert_atom(cfg.into()),
+                                    Some(pos) => {
+                                        let key = &cfg[..pos];
+                                        let value = cfg[pos + 1..].trim_matches('"');
```

Comment:
Why do we need to trim quotes?

Reply:
> Why do we need to trim quotes?

I pulled this section from: https://github.com/rust-analyzer/rust-analyzer/blob/master/crates/ra_project_model/src/lib.rs#L387

But ended up using a `key=value` string in the json that I was generating, instead of `key="value"`, which `rustc --print cfg -O` returns:

```
target_arch="x86_64"
target_endian="little"
...
```

Reply:
Ah, right, and the quotes are there because it's actually rust syntax : `#[cfg(target_os="linux")]`...

I guess I am fine with making them optional, although it does sound a bit iffy.

Reply:
I'll setup the import to assume that they're present (matching the output from `rustc`), and have parity with how the Cargo.toml project setup works:

https://github.com/rust-analyzer/rust-analyzer/blob/master/crates/ra_project_model/src/lib.rs#L387